pkgs/development/python-modules/treq/default.nix

```diff
@@ -33,13 +33,12 @@ buildPythonPackage rec {
     trial treq
   '';
 
-  doCheck = false;
-  # Failure: twisted.web._newclient.RequestTransmissionFailed: [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_get_server_certificate', 'certificate verify failed')]>]
+  patchPhase = ''
+    rm -fv src/treq/test/test_treq_integration.py
+  '';
 
   postBuild = ''
-    ${coreutils}/bin/mkdir -pv treq
-    ${coreutils}/bin/echo "${version}" | ${coreutils}/bin/tee treq/_version
-    cd docs && ${gnumake}/bin/make html && cd ..
+    tox -e docs
```

Comment:
do always document why you run commands like these because its not obvious at all what it does.